CoreAnalyticSchema/src/au/gov/asd/tac/constellation/graph/schema/analytic/concept/TemporalConcept.java

```diff
@@ -52,25 +53,25 @@ public String getName() {
     public static class VertexAttribute {
 
         public static final SchemaAttribute DATETIME = new SchemaAttribute.Builder(GraphElementType.VERTEX, ZonedDateTimeAttributeDescription.ATTRIBUTE_NAME, "DateTime")
-                .setDescription("The datetime at which this node occurred").setFormat("DDMMYYYYHHMMSS")
```

Comment:
I don't think the `setFormat` function actually works (a bug), but perhaps use the default Constellation format over this random format here, and move all the `setFormat`'s to new lines to make it clear they are there.

Reply:
Not sure what the default format is but I've moved setFormats down. I think this has been the format used for a while so I'm alright to leave the format as is (granted this does seem to be your code...)